codespell_lib/data/dictionary.txt

```diff
@@ -6997,6 +7348,12 @@ auto-negoziation->auto-negotiation
 auto-negoziations->auto-negotiations
 auto-realease->auto-release
 auto-realeased->auto-released
+autoamtic->automatic
+autoamtically->automatically
+autoattak->autoattack
```

Comment:
Is that a word? I have found the [AutoAttack](https://autoattackgames.com/) game studio, but that's about it.

Reply:
I think this is some kind of gaming related term.

Reply:
I see. Why not...

Reply:
I'm assuming that codespell might be of interest also to game developers. I agree that it's at least somewhat dubious, but on the other hand it doesn't hurt much to leave it in.